cs_upload.py

```diff
@@ -362,7 +458,12 @@ def main():
     parser.add_argument(
         "--verbose",
         action="store_true",
-        help="Trace or scan file to upload.",
+        help="Print out more detailed logging",
+    )
+    parser.add_argument(
+        "--insecure",
+        action="store_true",
+        help="Disable the certificate validation.",
```

Comment:
**suggestion**: Use `--no-check-certificate` since this is what we have in the OCaml cs-api tool.

Also, I would suggest using this name internally instead of `check_ca_cert` which implies we only check the CA cert, which is incorrect: curl also checks the whole chain and the validity of the server certificate (hostname + expiration + other properties).

Reply:
Changed!